src/oauth-manager/token-utils.ts

```diff
@@ -62,6 +63,53 @@ export function renderTokenCallback(instanceUrl: string, oauthReqInfo: string) {
                 font-size: 0.85rem;
                 color: #6c757d;
             }
```

Comment:
Can we make this html block more modular, currently it will be hard to debug any errors in this area. Maybe start by having separate html file and a separate css for this.